src/main/java/com/sobytylnik/ProfileController.java

```diff
@@ -0,0 +1,46 @@
+package com.sobytylnik;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import java.util.List;
+import java.util.Optional;
+
+
+@RestController
+public class ProfileController {
+
+    @Autowired
+    private ProfileRepository repository;
+
+    @RequestMapping(value="/profiles", method = RequestMethod.GET)

```

Comment:
According to the task, endpoint have to be `GET /api/profiles`, not `GET /profiles`